projects/1.16/assets/pneumaticcraft-repressurized/pneumaticcraft/patchouli_books/book/zh_cn/entries/machines/gas_lift.json

```
@@ -6,20 +6,20 @@
   "pages": [
     {
       "type": "text",
-      "text": "简而言之，$(item)气举/$就是一个泵。其使用$(l:base_concepts/pressure)压力/$减小某种流体的密度，以迫使其上升。$(p)提供压力后，其将向下伸出$(l:components/drill_pipe)气举钻管/$（这也是必需的），直到其接触到流体；此时气举将会泵取这一流体。这使得其很适合泵取$(l:base_concepts/oil)石油/$，因为其可能处于地层深处。"
+      "text": "简而言之，$(item)气举/$就是一个泵。其使用$(l:base_concepts/pressure)压力/$来减小流体的密度，从而迫使其上升。$(p)提供压力后，它会向下伸出$(l:components/drill_pipe)气举钻管/$（这也需要你提供给它），直到钻管接触到流体；这时气举才会泵取这一流体。这就使得气举非常适合泵取$(l:base_concepts/oil)石油/$，因为石油往往处于地层深处。"
```

```suggestion
      "text": "简而言之，$(item)气举/$就是一个泵。其使用$(l:base_concepts/pressure)压力/$来减小流体的密度，从而迫使其上升。$(p)提供压力后，它会向下伸出$(l:components/drill_pipe)气举钻管/$（需要自行提供），直到钻管接触到流体；这时气举才会泵取这一流体。这就使得气举非常适合泵取$(l:base_concepts/oil)石油/$，因为石油往往处于地层深处。"
```
不知道这样可不可以；或者“由玩家提供”？

---

没问题，pne手册的整体语言偏正式，需自行提供好一些。